.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data required**
+A clear and concise description of the data needed
+
+**Where does the data fit in the country profiles already uploaded**
```

Comment:
Shouldn't we also ask how the data should preferably be presented here?

Reply:
@kilemensi yes let me add this